sycl/include/CL/sycl/backend_types.hpp

```diff
@@ -10,12 +10,34 @@
 
 #include <CL/sycl/detail/defines.hpp>
 
+#include <iostream>
+
 __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 
 enum class backend : char { host, opencl, level0, cuda };
 
 template <backend name, typename SYCLObjectT> struct interop;
 
+inline std::ostream &operator<<(std::ostream &Out, backend be) {
+  switch (be) {
+  case backend::host:
+    Out << std::string("host");
+    break;
+  case backend::opencl:
+    Out << std::string("opencl");
+    break;
+  case backend::level0:
+    Out << std::string("level0");
```

Comment:
Since you are adding one more use of "level0" may I ask you to re-name all "level0" to "level-zero" ("level_zero" in enum)?

Reply:
FWIW, I am fine with the name change in a separate PR too.